apps/www/components/Pledge/Submit.js

```diff
@@ -610,151 +898,165 @@ class Submit extends Component {
-
-              if (
-                state.values.paymentMethod !== nextState.values.paymentMethod ||
-                state.values.paymentSource !== nextState.values.paymentSource
-              ) {
-                nextState.showErrors = false
-                nextState.errors = {}
+        <div {...styles.topMargin}>
+          <PaymentForm
+            key={me && me.id}
+            ref={this.paymentRef}
+            t={t}
+            loadSources={!!me || !!query.token}
+            accessToken={query.token}
+            requireShippingAddress={requireShippingAddress}
+            payload={{
+              id: this.state.pledgeId,
+              userId: this.state.userId,
+              total: this.props.total,
+              pfAliasId: this.state.pfAliasId,
+              pfSHA: this.state.pfSHA,
+            }}
+            context={packageName}
+            allowedMethods={paymentMethods}
+            userName={userName}
+            userAddress={userAddress}
+            addressState={addressState}
+            shippingAddressState={shippingAddressState}
+            syncAddresses={syncAddresses}
+            packageGroup={packageGroup}
+            setSyncAddresses={setSyncAddresses}
+            onChange={(fields) => {
+              const setPaymentMethod =
+                fields.values?.paymentMethod &&
+                fields.values.paymentMethod !== selectedPaymentMethod
+              if (setPaymentMethod) {
+                setSelectedPaymentMethod(fields.values.paymentMethod)
               }
+              this.setState((state) => {
+                const nextState = FieldSet.utils.mergeFields(fields)(state)
 
-              return nextState
-            })
-          }}
-          values={this.state.values}
-          errors={this.state.errors}
-          dirty={this.state.dirty}
-        />
-        <br />
-        <br />
+                if (
+                  setPaymentMethod ||
+                  state.values.paymentSource !== nextState.values.paymentSource
+                ) {
+                  nextState.showErrors = false
+                  nextState.errors = {}
+                }
+
+                return nextState
+              })
+            }}
+            values={{
+              ...this.state.values,
+              paymentMethod: selectedPaymentMethod,
+            }}
+            errors={this.state.errors}
+            dirty={this.state.dirty}
+          >
+            {stripeError && <ErrorMessage error={stripeError} />}
+            {
+              // Only render the browser API in case we're not using a browser payment API
+              !me &&
+                this.props.selectedPaymentMethod &&
+                !this.isStripeWalletPayment() && (
+                  <div {...styles.topMargin}>
+                    {contactPreface && (
+                      <div style={{ marginBottom: 40 }}>
+                        <P>{contactPreface}</P>
+                      </div>
+                    )}
```

Comment:
We should move this preface above the payment question:
<img width="795" alt="Screenshot 2022-03-18 at 11 08 19" src="https://user-images.githubusercontent.com/410211/158983593-189f1f85-24ad-4bee-9296-ff9e8a88ce52.png">

Reply:
![Uploading image.png…]()
Done